src/v2i-hub/ImmediateForwardPlugin/manifest.json

```diff
@@ -52,10 +52,20 @@
       "default": "0",
        "description": "The destination UDP server(s) and port number(s) on the V2X radio for all messages specified by Messages_Destination_4."
     },
+    {
+      "key": "TxMode",
+      "default": "CONT",
+      "description": "Transmission channel mode (continuous or alternating). Allowed values: CONT, ALT."
+    },
     {
       "key": "signMessage",
       "default": "0",
-      "description": "0 if False and 1 if True indicating whether should sign outgoing J2735 messages."
+      "description": "0 if False and 1 if True, indicating whether Active File will indicate to sign outgoing J2735 messages."
+    },
+    {
+      "key": "EnableHSM",
+      "default": "0",
+      "description": "0 if False and 1 if True, indicating whether HSM is connected and will sign outgoing J2735 messages."
```

Comment:
Is True HSM connected and sign the messages? What does HSM stand for?

Reply:
Yeah, true means V2X Hub would try to sign the messages using a Hardware Security Module (only worked with one specific module).